bundles/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/InfluxDBPersistenceService.java

```diff
@@ -190,60 +142,60 @@ public String getLabel(@Nullable Locale locale) {
     @Override
     public Set<PersistenceItemInfo> getItemInfo() {
         if (checkConnection()) {
-            return influxDBRepository.getStoredItemsCount().entrySet().stream()
-                    .map(entry -> new InfluxDBPersistentItemInfo(entry.getKey(), entry.getValue()))
+            return influxDBRepository.getStoredItemsCount().entrySet().stream().map(InfluxDBPersistentItemInfo::new)
                     .collect(Collectors.toUnmodifiableSet());
         } else {
-            logger.info("getItemInfo ignored, InfluxDB is not yet connected");
+            logger.info("getItemInfo ignored, InfluxDB is not connected");
             return Set.of();
         }
     }
 
     @Override
     public void store(Item item) {
-        store(item, item.getName());
+        store(item, null);
     }
 
     @Override
     public void store(Item item, @Nullable String alias) {
         if (checkConnection()) {
             InfluxPoint point = itemToStorePointCreator.convert(item, alias);
             if (point != null) {
-                logger.trace("Storing item {} in InfluxDB point {}", item, point);
-                influxDBRepository.write(point);
+                try {
+                    influxDBRepository.write(point);
+                    logger.trace("Stored item {} in InfluxDB point {}", item, point);
+                } catch (UnexpectedConditionException e) {
+                    logger.warn("Failed to store item {} in InfluxDB point {}", point, item);
+                }
             } else {
-                logger.trace("Ignoring item {} as is cannot be converted to an InfluxDB point", item);
+                logger.trace("Ignoring item {}, conversion to a InfluxDB point failed.", item);
```

Comment:
This would revert a typo fix from https://github.com/openhab/openhab-addons/pull/13819/files#diff-05fde9430a9ef73b9401cdb1375bc43bd99b501679e037d24e67eb5717208bf4 in #13819
```suggestion
                logger.trace("Ignoring item {}, conversion to an InfluxDB point failed.", item);
```